changelog.txt

```diff
@@ -15,6 +15,10 @@ that repo.
 
 ## New Scripts
 
+- `max-wave`: dynamically limit the next immigration wave, can be set to repeat
+- `pop-control`: persistent per fortress population cap, `hermit`, and `max-wave` management
+- `assign-minecarts`: assign minecarts to hauling routes that don't have one
```

Comment:
wait, didn't we fix this already? the `assign-minecarts` line is from the previous release.

Reply:
I might not be merging as well as I thought.